src/Sdk/Subscriptions/Subscriptions.ts

```diff
@@ -0,0 +1,58 @@
+import ApiClient from '../ApiClient';
+import { ContactType, EmailSubscription, Newsroom } from '../../types';
+import {
+    NewsroomSubscribeRequest,
+    NewsroomUnsubscribeRequest,
+    UpdateNewsroomUnsubscribeDetailsRequest,
+} from './types';
+import routing from '../routing';
+
+type NewsroomId = Newsroom['uuid'] | Newsroom['id'];
+
+export default class Subscriptions {
+    private readonly apiClient: ApiClient;
+
+    constructor({ apiClient }: { apiClient: ApiClient }) {
+        this.apiClient = apiClient;
+    }
+
+    public async subscribeToNewsroom(
+        newsroomId: NewsroomId,
+        payload: NewsroomSubscribeRequest<ContactType.PERSON | ContactType.ORGANISATION>,
+    ): Promise<EmailSubscription> {
```

Comment:
> Also check how I specified type of payload parameter of subscribeToNewsroom method.

We can make it even better :)


```suggestion
    public async subscribeToNewsroom<Type extends ContactType>(
        newsroomId: NewsroomId,
        payload: NewsroomSubscribeRequest<Type>,
    ): Promise<EmailSubscription> {
```